### PR TITLE
Bytecode removal from Java GLV

### PR DESCRIPTION
**Pull request by vkagamlyk**
Java GLV will no longer send requests as Bytecode.

Feature tests `GraphBinaryRemoteFeatureTest` switched to run with `gremlin-lang` due to minor incompatibilities with `gremlin-groovy`.

### CODE REVIEW

File: gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/util/BytecodeHelper.java [hunk 1]
```diff
@@ -177,7 +175,7 @@
 import java.util.stream.Stream;
 
 /**
- * Utility class for parsing {@link Bytecode}.
+ * Utility class for parsing {@link GremlinLang}.
```
Comment (xiazcy) [inline gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/util/BytecodeHelper.java:178]: nit: don't forget classname update

File: gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/jsr223/GremlinLangScriptEngine.java [hunk 2]
```diff
@@ -79,7 +79,7 @@ public GremlinScriptEngineFactory getFactory() {
      * Bytecode is evaluated by the {@link JavaTranslator}.
      */
     @Override
-    public Traversal.Admin eval(final Bytecode bytecode, final Bindings bindings, final String traversalSource) throws ScriptException {
+    public Traversal.Admin eval(final GremlinLang bytecode, final Bindings bindings, final String traversalSource) throws ScriptException {
```
[the remark this answers is not in the document]
Comment (vkagamlyk) [reply gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/jsr223/GremlinLangScriptEngine.java:82]: I hope to remove this altogether in one of the next PRs, 'GremlinScriptEngineFactory' will handle only gremlin scripts.

**Comment by xiazcy** [comment]
VOTE +1
Thanks for putting together the work! Looking forward to the rest of the changes.

**Comment by vkagamlyk** [comment]
VOTE +1

**Comment by kenhuuu** [comment]
VOTE +1

File: gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/util/BytecodeHelper.java [hunk 3]
```diff
@@ -177,7 +175,7 @@
...
  *
```
Comment (Cole-Greer) [inline gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/util/BytecodeHelper.java:179]: Should this class be renamed to GremlinLangHelper?

Comment (vkagamlyk) [reply gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/util/BytecodeHelper.java:179]: it should be removed later

**Comment by Cole-Greer** [comment]
VOTE +1

**Pull request merged**